.github/workflows/prod.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Push to Github Packages'
+
+on:
+    pull_request:
+        types: closed
+        branches:
+            - master
```

Comment:
We're using "main" here in place of "master". https://www.bbc.com/news/technology-53050955